ui/src/app/app-routing.module.ts

```diff
@@ -77,7 +82,10 @@ const routes: Routes = [
           component: WorkspaceNavBarComponent,
           data: {
             title: 'View Workspace Details',
-            breadcrumb: 'Param: Workspace Name'
+            breadcrumb: {
+              value: 'Param: Workspace Name',
+              intermediate: false
```

Comment:
opt_nit: I would probably also just omit the value instead of false just to make this a bit more readable (code checking `!breadcrumb.intermediate` will work either way, or I would use `!!breadcrumb.intermediate` if you need to coerce to boolean)